client/verta/verta/_internal_utils/http_session_utils.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+""" Utilities for working with request.Session objects, and handling of retries """
+
+# TODO: Unit test for prediction_input_unpack method
+# TODO: Unit test for prediction_io_cleanup method
+import requests
+
+from requests import Session
+from requests.adapters import HTTPAdapter
+from typing import Dict, Optional, Set
+from urllib3.util import Retry
```

Comment:
```suggestion
from typing import Dict, Optional, Set

import requests
from requests import Session
from requests.adapters import HTTPAdapter
from urllib3.util import Retry
```

[PEP 8](https://peps.python.org/pep-0008/#imports):

>  Imports should be grouped in the following order:
> 
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
> 
> You should put a blank line between each group of imports.